bodhi/tests/server/scripts/test_check_policies.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+# Copyright ® 2017 Red Hat, Inc.
+#
+# This file is part of Bodhi.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""This module contains tests for the bodhi.server.scripts.check_policies module."""
+
+from click import testing
+from mock import patch
+
+from bodhi.server import models
+from bodhi.server.scripts import check_policies
+from bodhi.tests.server.base import BaseTestCase
+from bodhi.server.config import config
+
+
+class TestCheckPolicies(BaseTestCase):
+    """This class contains tests for the check_policies() function."""
+    @patch.dict(config, [('greenwave_api_url', 'http://domain.local')])
+    def test_policies_satisfied(self):
+        """Assert correct behavior when the policies enforced by Greenwave are satisfied"""
+        runner = testing.CliRunner()
+        update = self.db.query(models.Update).all()[0]
+        update.status = models.UpdateStatus.testing
+        self.db.commit()
+        with patch('bodhi.server.scripts.check_policies.greenwave_api_post') as mock_greenwave:
+            expected_json = {
```

Comment:
The name "expected" makes it sound like we are going to use it in an assertion, but really it's just mock data. Why not name it "greenwave_response" or similar?